defaults/main.yml

```diff
@@ -287,6 +299,21 @@ docker__pki_crt: 'default.crt'
 #
 # Name of the private key file used by Docker.
 docker__pki_key: 'default.key'
+
+                                                                   # ]]]
+# .. envvar:: docker__private_registry_keys [[[
+#
+# Dictionary with private registry name and keys for
+# secure client-server communication.
+#
+# .. code-block:: yaml
+#    :linenos:
+#
+#    docker__private_registry_keys:
+#      'myregistry1.domain.com': '/etc/pki/realm/domain/CA.crt'
+#      'myregistry2domain.com:5000': '/etc/pki/realm/domain/CA.crt'
+docker__private_registry_keys: {}
```

Comment:
This doesn't seem to be used anywhere.

If you add the root CA certificate on the host to the system CA store, does it work without specifying the CA certificate in question in Docker itself?